src/main/webapp/web.xml

```diff
@@ -86,7 +86,7 @@
     </servlet>
     <servlet-mapping>
         <servlet-name>allManufacturers</servlet-name>
-        <url-pattern>/manufacturers/all</url-pattern>
```

Comment:
you forgot to remove these /all from DeleteCarController, DeleteDriverController and DeleteManufacturerController